app/models/user.rb

```diff
@@ -11,8 +11,24 @@ class User < ApplicationRecord
   has_many :post_comments, through: :comments, class_name: 'Post'
   has_many :likes
   has_many :post_likes, through: :likes
+  has_many :friendships
+  has_many :inverse_friendships, class_name: 'Friendship', foreign_key: 'friend_id'
   validates :first_name, presence: true, length: { minimum: 2 }
   validates :last_name, presence: true, length: { minimum: 2 }
   validates :date_birth, presence: true
   validates :gender, presence: true, inclusion: { in: %w[male female] }
+
```

Comment:
- [ ] We should also have a method in the user model to show the posts made by the user and any of his friends, so it's easier to call in the views without having to use enumerable (use active record syntaxis)